.github/workflows/maven.yml

```diff
@@ -15,23 +15,43 @@ jobs:
         jdk: ['8','11']
         os: [ubuntu-latest]
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK ${{ matrix.jdk }}
-      uses: actions/setup-java@v1
-      with:
-        java-version: ${{ matrix.jdk }}
-    - name: Cache Maven packages
-      uses: actions/cache@v2
-      with:
-        path: ~/.m2
-        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
-        restore-keys: ${{ runner.os }}-m2
-    - name: Build with Maven
-      run: mvn -B -U verify
-    - name: Running Maven GUI tests
-      run: |
-        sudo apt-get install fluxbox
-        export DISPLAY=':99.0'
-        Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
-        fluxbox &> ~/fluxbox.log &
-        mvn -B verify -Pwith-gui-tests
+      - uses: actions/checkout@v2
+      - name: Set up JDK ${{ matrix.jdk }}
+        uses: actions/setup-java@v2
+        with:
+          distribution: temurin
+          java-version: ${{ matrix.jdk }}
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Build with Maven
+        run: mvn -B -U verify
+      - name: Running Maven GUI tests
+        run: |
+          sudo apt-get install fluxbox
+          export DISPLAY=':99.0'
+          Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+          fluxbox &> ~/fluxbox.log &
+          mvn -B verify -Pwith-gui-tests
+  deploy:
+    if: github.ref == 'refs/heads/main'
```

Comment:
Should be: ```if: contains( github.ref, 'master')```
See https://docs.github.com/en/actions/migrating-to-github-actions/migrating-from-gitlab-cicd-to-github-actions#condition-and-expression-syntax

Reply:
Fixed on master